docker-compose.yml

```diff
@@ -259,6 +259,8 @@ services:
     restart: always
     volumes:
       - ${DATA_DIR:-./data}/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_HOST_AUTH_METHOD: "trust"
```

Comment:
Should we set `POWERUSER_PASSWORD` instead?

Reply:
the change made equals exactly how it was before. I think it does not make much difference for now